style.css

```diff
@@ -493,3 +504,305 @@ form ul li button:hover {
+    text-decoration: none;
+    width: 63px;
+    height: 20px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 20px;
+    color: #fff;
+  }
+
+  .hero-section {
+    width: 100%;
+    height: 45rem;
+    background: #1c1a19;
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    padding-bottom: 10rem;
+    background-image: url("./images/Illustration-header.png");
+    background-repeat: no-repeat;
+    background-position: top right;
+  }
+
+  .hero-section img {
+    width: 80%;
+    height: 80%;
+  }
+
+  .hero-left {
+    background-image: none;
+    height: 0;
+  }
+
+  /* ----------------------- to remove stylelint error used this extra comment----------- */
+  form .contact-ul {
+    display: flex;
+    flex-direction: row;
+    width: 88%;
+    gap: 2.5rem;
+  }
+
+  .hero-left ul {
+    display: flex;
+    flex-direction: column;
+    margin-top: 8rem;
+    width: 5%;
+    height: 30rem;
+    order: 1;
+  }
+
+  .contact-right li {
+    display: flex;
+    justify-content: flex-start;
+  }
+
+  .hero-left ul li {
+    margin-top: 1.5rem;
+  }
+
+  .hero-right {
+    display: flex;
+    flex-direction: column;
+    width: 95%;
+    height: 50rem;
+    order: 2;
+    margin-left: 10rem;
+    margin-top: 8rem;
+    padding-top: 2rem;
+    margin-bottom: 30rem;
+  }
+
+  .hero-right h1 {
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+    color: #f55800;
+  }
+
+  .hero-right p {
+    width: 50%;
+  }
+
+  .work-details-row1 {
+    width: 90%;
+    display: flex;
+    flex-direction: row;
+    margin-top: 1rem;
+    padding: 0 1rem;
+  }
+
+  .work-heading {
+    display: flex;
+    height: 10rem;
+    flex-direction: row;
+    align-items: center;
+    width: 90%;
+  }
+
+  .work-heading h4 {
+    width: 25rem;
+    padding-left: 2rem;
+    margin-bottom: 8rem;
+  }
+
+  .work-heading hr {
+    width: 65rem;
+    height: 0;
+  }
+
+  .work-details {
+    width: 90%;
+    height: auto;
+  }
+
+  .work-details-row1 img {
+    width: 50%;
+    height: 300px;
+    background-image: url("./images/big row1 img.png");
+    background-repeat: no-repeat;
+    background-position: top;
+    padding-left: 0;
+  }
+
+  .details-right1 {
+    display: flex;
+    flex-direction: column;
+    width: 60%;
+    padding-left: 2rem;
+    padding-right: 0;
+  }
+
+  .card-work {
+    width: 100%;
+    height: auto;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr;
+    gap: 1rem;
+    padding: 2rem 0 6rem 0;
+    align-items: center;
+  }
+
+  .work {
+    width: 100%;
+    height: 28rem;
+    background: linear-gradient(180.45deg, rgba(71, 70, 70, 0) 0.75%, rgba(77, 75, 75, 0.9) 61.94%), url("images/Bg_ImgCard.png");
+    background-repeat: round;
+    float: left;
+    transition: 0.5s ease-in-out;
+  }
```

Comment:
- [ ] As suggested by the previous reviewer, ON HOVER OF BUTTON when the project is hovered on with the mouse the description and dark opacity disappear and a See Project button appears at the bottom of the image and takes the whole image width 👍🏿. On hover your project should look like the card that has the arrow, please achieve this functionality for all cards in this section 👍🏿. 
   - `IMPORTANT NOTE: `  The description should also disappear. Consider your project in the video attached.
- [ ] I have also noticed that there is still the horizontal scroll effect 😌 . The experience of scrolling horizontally can make the screen judder and create a headache-inducing feeling.  I kindly suggest that you remove the horizontal effect appearing. 

- The video 😎 

https://user-images.githubusercontent.com/99394374/214850631-e7ea786d-9f0b-46f3-b497-a5f86bae6384.mp4